src/main/java/codeu/model/data/User.java

```diff
@@ -14,53 +14,250 @@
 
 package codeu.model.data;
 
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
 import java.time.Instant;
 import java.util.UUID;
+
 import java.util.*;
 
+enum OccupationType{
```

Comment:
Bring this inside the User class. Generally the best practice for Java is that only the class should be available at the file level. Once its nested inside the class you can reference it via User.OccupationType.